examples/terraform/azure/variables.tf

```
@@ -69,3 +69,12 @@ variable "worker_vm_size" {
   default     = "Standard_B2s"
 }
 
+variable "control_plane_vm_count" {
+  description = "VM Size for worker machines"
```

We should adjust the description to make it clear that it's about the number of virtual machines and not about the instance size/type. For example:

```suggestion
  description = "Number of control plane instances"
```